src/app/constants/concerts-upcoming-items.constants.ts

```diff
@@ -29,21 +28,22 @@ export const CONCERTS_UPCOMING_ITEMS: readonly ConcertUpcomingItem[] = [
     description: 'A mise után a kórussal közös koncertünkön Pergolesi csodálatos Stabat Materét hallhatja a közönség.',
     inviteImageSource: 'pergolesi',
     mapUrl: 'https://goo.gl/maps/23nARtjyrZbkHcto6?coh=178572&entry=tt',
+    facebookEventUrl: null,
   },
 ];
 
-const lastConcert: ConcertUpcomingItem = {
-  number: 1,
-  imageDesktopSource: 'janos-passio',
-  imageMobileSource: 'janos-passio-mobile',
-  imageAltTranslationKey: 'BVKZ.CONCERTS_CONTAINER__COMPONENT.CONCERTS_UPCOMING__IMAGE_ALT_TEXT',
-  title: 'Johann Sebastian Bach: János-passió, BWV 245',
-  date: '2023. március 18., 19:30',
-  dateCode: '2023-03-18T19:30',
-  concertLengthInMinutes: 120,
-  place: 'Belvárosi Ferences Templom, Ferenciek tere 9.',
-  description:
-    'Nehezen tudunk elfogódottság nélkül közelíteni ehhez a fantasztikus zeneműhöz, így nagy megtiszteltetés számunkra, hogy harmadik alkalommal játszhatjuk Bach János-passióját a Pesti Ferences Templomban.',
-  inviteImageSource: 'janos-passio-invite',
-  mapUrl: 'https://goo.gl/maps/W9LXZQKcwSJkYmKh9',
-};
+// const lastConcert: ConcertUpcomingItem = {

```

Comment:
commentet nem hagyunk a kodban, toroljuk es  a vele jaro folosleges forditasi kulcsokat is (ha van ilyen)

Reply:
de ezt te tetted oda, arra az esetre, ha nem lenne közelgő koncert, hogy akkor is tudjuk, hogy hogy nézett ki.

Reply:
töröltem az egész fájlt